src/app/GitUI/CommandsDialogs/FormVerify.LostObject.cs

```diff
@@ -78,6 +78,30 @@ private LostObject(LostObjectType objectType, string rawType, ObjectId objectId)
                 ObjectId = objectId ?? throw new ArgumentNullException(nameof(objectId));
             }
 
+            public static string[] GetCommitsMetadata(IGitModule module, string commits)
```

Comment:
How is this method used?

Reply:
I have added a comment (and changed a little the interface)